index.html

```diff
@@ -582,13 +582,13 @@ <h3>Use Cases and Requirements</h3>
         </p>
 
         <p>
-Digital proof mechanisms, a subset of which are digital signatures, are required
-to ensure the protection of a <a>verifiable credential</a>. Having and
+<a>Verifiable credential</a> and <a>verifiable presentation</a> MUST be protected using a digital proof
+mechanism such as digital signatures. Having and
 validating proofs, which may be dependent on the syntax of the proof
```

Comment:
```suggestion
verifying proofs, which may be dependent on the syntax of the proof
```
I'd like to avoid the verification/validation confusion here. The English isn't *wrong* but proofs are used for verification, so that seems a less confusing word choice. Also, I know that wasn't part of this PR, but let's fix it.